src/components/Footer.js

```diff
@@ -1,44 +1,63 @@
 import React from "react"
-import Title from "./Title"
-import { graphql, useStaticQuery } from "gatsby"
-import { Box, Text } from "@chakra-ui/core"
-import Img from "gatsby-image"
+import { Box, PseudoBox, Heading } from "@chakra-ui/core"
 
-const Footer = () => {
-    const { organizers } = useStaticQuery(graphql`
-        query {
-            organizers: file(relativePath: { eq: "organizers.jpg" }) {
-                childImageSharp {
-                    fluid(maxWidth: 2000, maxHeight: 700) {
-                        ...GatsbyImageSharpFluid_withWebp
-                    }
-                }
-            }
-        }
-    `)
-    return (
+export default () => (
+    <Box px="10vw" py="75px" color="blue.500">
         <Box
-            textAlign="center"
             display="flex"
-            flexDirection="column"
+            flexDirection={["column", "column", "row", "row"]}
+            justifyContent="center"
             alignItems="center"
         >
-            <Box px="10vw">
-                <Title
-                    sub="Lorem ipsum dolor"
-                    main="Lorem ipsum dolor sit amet"
-                />
-                <Text color="gray" maxWidth="600px" mt="25px" mb="100px">
-                    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
-                    do eiusmod tempor incididunt ut labore et dolore magna
-                    aliqua.
-                </Text>
+            <Box display="flex" justifyContent="center" alignItems="center">
+                <PseudoBox mx="25px" _hover={{ color: "blue.300" }}>
```

Comment:
again, check across screensizes (use polypane or something) for responsiveness, and change `mx` with breakpoints where appropriate